schema.sql

```diff
@@ -26,75 +25,75 @@ CREATE TABLE dates (
 	PRIMARY KEY (id),
 	FOREIGN KEY (apartment_id)
```

Comment:
recommend removing foreign key (apartment_id) because apartmentid was removed

Reply:
great catch here, this will be caught by the foreign key constraint if the apartment is invalid.  no need for the select statement